src/components/Tag/native/Tag.js

```diff
@@ -51,20 +55,18 @@ const Tag = ({ style, color, kind, size, children }) => (
 );
 
 Tag.propTypes = {
```

Comment:
Make sure, propTypes and defaultProps are as per guidelines too. Tag's need to accept the shape `sharpEdged` as well. Take a look at the web version of the Tag component for the props.